.github/workflows/helm-build.yml

```diff
@@ -3,28 +3,71 @@ name: Helm CI - TIP WLAN Cloud Master
 on: 
   push:
     branches: [ master ]
+    tags: [ "v*" ]
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-        with:
-          ssh-key: ${{ secrets.GH_AUTOMATION_KEY }}
-          submodules: true
-      - name: Login to TIP Docker registry 
-        uses: azure/docker-login@v1
-        with:
-          login-server: tip-tip-wlan-cloud-docker-repo.jfrog.io
-          username: build-pipeline
-          password: ${{ secrets.DOCKER_REPO_PASSWORD }}
       - name: Login to TIP Helm chart registry
         run: helm repo add tip-wlan-cloud-helm-virtual-repo https://tip.jfrog.io/artifactory/tip-wlan-cloud-helm-virtual-repo --username build-pipeline --password ${{ secrets.HELM_REPO_PASSWORD }}
       - name: Build tip-wlan chart file
-        run: tar -czf tip-wlan.tgz tip-wlan
+        run: |
+          if [[ "${{ github.ref }}" == "refs/tags/"* ]]; then
+            PACKAGE_OPTS="--version ${GITHUB_REF#refs/tags/v}"
+          else
+            PACKAGE_OPTS=""
+          fi
+          helm package $PACKAGE_OPTS -u tip-wlan
+      - name: Store chart as artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: helm-chart
+          path: tip-wlan-*.tgz
       - name: Upload tip-wlan chart to the TIP helm registry
-        run: curl -ubuild-pipeline:${{ secrets.HELM_REPO_PASSWORD }} -T tip-wlan.tgz "https://tip.jfrog.io/artifactory/tip-wlan-cloud-helm-repo/tip-wlan.tgz"
-      - name: Verify that chart was uploaded successfully
         run: |
-          helm repo update
-          helm search repo tip
\ No newline at end of file
+          if [[ "${{ github.ref }}" == "refs/tags/"* ]]; then
+            curl -ubuild-pipeline:${{ secrets.HELM_REPO_PASSWORD }} -T tip-wlan-${GITHUB_REF#refs/tags/v}.tgz "https://tip.jfrog.io/artifactory/tip-wlan-cloud-helm-repo/tip-wlan-${GITHUB_REF#refs/tags/v}.tgz"
+          else
+            curl -ubuild-pipeline:${{ secrets.HELM_REPO_PASSWORD }} -T tip-wlan-*.tgz "https://tip.jfrog.io/artifactory/tip-wlan-cloud-helm-repo/tip-wlan-master.tgz"
+          fi
+
+  release:
+    runs-on: ubuntu-latest
+    needs: [ build ]
+    if: startsWith(github.ref, 'refs/tags/')
+    steps:
+      - uses: actions/checkout@v2
+      - name: setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+      - name: create RELEASE.md
+        run: |
+          cat << EOF > RELEASE.md
+          ## Installation
+
+          \`\`\`
+          $ helm repo add tip-wlan-cloud-helm-virtual-repo https://tip.jfrog.io/artifactory/tip-wlan-cloud-helm-virtual-repo --username <USERNAME> --password <PASSWORD>
+          $ helm install tip-wlan-cloud-helm-virtual-repo/tip-wlan --version ${GITHUB_REF#refs/tags/v}
```

Comment:
I'm not sure that I like the fact that the installation procedure is hard-coded into the build pipeline, but to be honest, I can't think of a better way? Maybe having a separate script, something like `install.sh` and then just pasting it's content here? Still not convinced it's better, though.

Reply:
why not put this into extract-changelog-entry.py? this way you can test this flow locally and you have an advantage of not needing this step?

I'd probably also create a container with python and keepachangelog already installed for the same reason. easier for folks to test\follow this locally

Reply:
@leonidlm Removed this section cause users need to preconfigure the environment anyway before being able to install. We can think about adding those installation instructions once the chart is actually usable in the "normal" way.

@4c74356b41 Removed this section for now. Don't see the need for a Docker image just for this purpose ATM. Don't expect anyone to execute this script locally prior to making a release.